tests/testthat/test-read_from_created_wb.R

```diff
@@ -40,39 +40,47 @@ test_that("Reading NAs and NaN values", {
   fileName <- file.path(tempdir(), "NaN.xlsx")
   na.string <- "*"
 
+  wb <- wb_workbook()
+
   ## data
-  a <- data.frame(
+  A <- data.frame(
```

Comment:
I assume this test was copied over from `openxlsx` (at least I hope nobody here uses `c` as object name) and shortened when we did not support all features. Now that we support more features, we might want to add the missing pieces.